tsconfig.json

```diff
@@ -3,31 +3,32 @@
   "compilerOptions": {
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
+    "declaration": true,
+    "declarationMap": true,
+    // "erasableSyntaxOnly": true,
```

Comment:
I'd like to implement this soon. The only rule we violate is the presence of enums.